dashboard/src/components/authentication/login/login.tsx

```diff
@@ -6,9 +6,8 @@ import CardContent from '@material-ui/core/CardContent';
 import CardActions from '@material-ui/core/CardActions';
 import Button from '@material-ui/core/Button';
 import CardHeader from '@material-ui/core/CardHeader';
-import Link from '@material-ui/core/Link';
-import Navbar from '../../common/navbar/navbar';
 import Footer from '../../common/footer/footer';
+import { Link } from 'react-router-dom';
```

Comment:
Could you please swap line 9 and line 10 coz we have to group custom made imports and others separately.